### PR TITLE
add recipe for larray-editor package

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

**Comment by alixdamman** [comment]
@gdementen OK with this PR?

File: recipes/larray-editor/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+"$PYTHON" setup.py install || exit 1
```
Comment (gdementen) [inline recipes/larray-editor/build.sh]: In https://github.com/ContinuumIO/menuinst/wiki/Menus-in-Conda-Recipes there are instructions for menus on Linux & MacOs. Did you try those?

File: recipes/larray-editor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "larray-editor" %}
+{% set version = "0.26" %}
```
Comment (gdementen) [inline recipes/larray-editor/meta.yaml]: Shouldn't that be updated to 0.26.1 (or even 0.27)?

File: recipes/larray-editor/build.sh (hunk 1 above)
Comment (alixdamman) [reply recipes/larray-editor/build.sh]: I'll try again as soon as possible.

File: recipes/larray-editor/meta.yaml (hunk 2 above)
Comment (alixdamman) [reply recipes/larray-editor/meta.yaml]: It would be nice to test the update to version 0.27 with the new `releaser` (when it's ready).

Comment (alixdamman) [reply recipes/larray-editor/meta.yaml]: But OK to update current meta.yaml with version 0.26.1

File: recipes/larray-editor/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/larray-project/larray-editor/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (mwcraig) [inline recipes/larray-editor/meta.yaml]: I'm pretty sure that if you include a script here then the separate build script will not run...given that you have build steps beyond setup.py, you should probably delete this line.

File: recipes/larray-editor/bld.bat [hunk 4]
```diff
@@ -0,0 +1,13 @@
+"%PYTHON%" setup.py install
```
Comment (mwcraig) [inline recipes/larray-editor/bld.bat]: Please add these args: `--single-version-externally-managed --record record.txt`

File: recipes/larray-editor/build.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+if [ $OSX_ARCH ]
+then
+    cp "$RECIPE_DIR/larray-editor.json" "$PREFIX/Menu"
+    cp "$SRC_DIR/larray_editor/images/larray.png" "$PREFIX/Menu"
+else
+    cp "$RECIPE_DIR/larray-editor.json" "$PREFIX/Menu"
+    cp "$SRC_DIR/larray_editor/images/larray.png" "$PREFIX/Menu"
+fi
+
+"$PYTHON" setup.py install || exit 1
```
Comment (mwcraig) [inline recipes/larray-editor/build.sh]: Please add these args: `--single-version-externally-managed --record record.txt`

File: recipes/larray-editor/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - larray_editor
+
+about:
+  home: https://github.com/larray-project/larray-editor
+  license: GPL-3.0
+  license_family: GPL
+  license_file: LICENSE
```
Comment (mwcraig) [inline recipes/larray-editor/meta.yaml]: I'm a little surprised this works...I thought the path to the recipe had to be included, but maybe not:
https://conda-forge.org/docs/meta.html#packaging-the-license-manually

**Comment by mwcraig** [comment]
Thanks, @alixdamman, merging...

**Comment by gdementen** [comment]
@mwcraig Thanks for the review & merge !

**Comment by alixdamman** [comment]
@mwcraig Thanks for the review.

**Pull request merged**